### DISCUSSION TITLE
Official Jellyfin shirt

### DISCUSSION THREAD

**Discussion by Weels92** [jellyfin/jellyfin#10171] [General] [answered]
Good morning,

Is there a place where I can buy an official Jellyfin T-shirt/Sweat?

**Comment by nielsvanvelzen** [comment, marked as answer]
We have a shop on Redbubble maintained by @joshuaboniface:
https://www.redbubble.com/people/joshuaboniface/shop

There is a 1% profit margin that is donated directly to our Open Collective, of which expenses like server infrastructure is paid:
https://opencollective.com/jellyfin

**Comment by Weels92** [comment]
Great, thx for your response !
